client/lib/Screens/login/login_screen.dart

```diff
@@ -0,0 +1,190 @@
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/material.dart';
+
+class LoginScreen extends StatelessWidget {
+  const LoginScreen({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Color(0xffFFFFFF),
+      body: ListView(
+        padding: EdgeInsets.symmetric(
+            horizontal: MediaQuery.of(context).size.width / 8),
+        children: const [
+          Menu(),
+          Body(),
+        ],
+      ),
+    );
+  }
+}
+
+class Menu extends StatelessWidget {
+  const Menu({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.symmetric(vertical: 30),
+      child: Row(
+        mainAxisAlignment: MainAxisAlignment.spaceBetween,
+        children: [
+          Row(
+            mainAxisAlignment: MainAxisAlignment.start,
+            children: [
+              Image.asset(
+                'images/logo-s.png',
+                width: 41,
+                height: 65,
+              )
+            ],
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class Body extends StatelessWidget {
+  const Body({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        SizedBox(
+          width: 380,
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              Text(
+                '   Sign In to the\nCommunity Portal',
```

Comment:
Do you plan to use a translation service?
Hardcoded sentences are not a good behavior

Reply:
Not currently, what should I do than?

Reply:
@g123k  what should I do?

Reply:
@g123k  suggestions please.

Reply:
Hello Sumit

You can have a look here 
https://docs.flutter.dev/development/accessibility-and-localization/internationalization
Hope it might help you out

Reply:
Thanks @AshAman999

Reply:
> Do you plan to use a translation service? 
currently not , might be in future updates .

Reply:
Please ask @teolemon how to use Crowdin (the same as Smoothie)

Reply:
It's as simple as tweaking the config file in the PR, with the path to the translation file (typically *.arb for flutter)

Reply:
https://github.com/openfoodfacts/community-portal/pull/25